client/src/modules/stock/lots/registry.js

```diff
@@ -18,6 +18,7 @@ function StockLotsController(Stock, Notify,
   var vm = this;
 
   var cacheKey = 'lot-grid';
+  var filterKey = 'lot';
```

Comment:
If possible, this would also clean up the other file that you use this pattern - stock movements.